src/Support/OMOptions.cpp

```diff
@@ -14,6 +14,8 @@
 
 #include "src/Support/OMOptions.hpp"
 
+using namespace onnx_mlir;
```

Comment:
Reminder to update copyright year as well from 2021 to 2022.

Reply:
Done. Thanks!